### PR TITLE
Create new app_info page

### PR DESCRIPTION
**Pull request by moollaza**
## Description
Duplicate of exisiting /app page, but without the footer.

**Note:**  This depends on internal CSS changes

/cc @brindy

### CODE REVIEW

File: share/site/duckduckgo/meta/app_info.tx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<: include "meta/title.tx" { title => l('Privacy, simplified. — DuckDuckGo Browser Extension & Mobile App') } :>
```
Comment (moollaza) [inline share/site/duckduckgo/meta/app_info.tx]: This is just a copy of `app.tx`

File: share/site/duckduckgo/app_info.tx [hunk 2]
```diff
@@ -0,0 +1 @@
+<div class="js-addons-page no-atb"></div>
```
Comment (moollaza) [inline share/site/duckduckgo/app_info.tx]: Copy of app.tx, with a new `no-atb` class, which is used by some internal CSS

File: lib/DDG/Publisher/Site/Duckduckgo/Root.pm [hunk 3]
```diff
@@ -307,6 +307,13 @@ sub pages {{
 		js_page_type => 'Addons',
 		copyright_footer => 1,
 	},
+	app_info => sub {
+		%page_defaults,
+		no_hero_header_icon => 1,
+		js_page_type => 'Addons',
+		about_footer => 0,
+		copyright_footer => 0,
```
Comment (moollaza) [inline lib/DDG/Publisher/Site/Duckduckgo/Root.pm]: Main difference here is the disabled footers. I'm re-using the `Addons` page type so the JS for the app page runs on this page as well.

**Comment by MariagraziaAlastra** [comment]
👍

**Pull request merged**